### PR TITLE
Update EIP-4626: New solmate url

### PR DESCRIPTION
**Pull request by transmissions11**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by transmissions11** [comment]
oof

**Comment by SamWilsn** [comment]
> oof

image

File: EIPS/eip-4626.md [hunk 1]
```diff
@@ -572,7 +572,7 @@ For production implementations of Vaults which do not use ERC-4626, wrapper adap
 
 ## Reference Implementations
 
-See [Solmate ERC4626](https://github.com/Rari-Capital/solmate/blob/main/src/mixins/ERC4626.sol):
+See [Solmate ERC4626](https://github.com/transmissions11/solmate/blob/main/src/mixins/ERC4626.sol):
```
Comment (MicahZoltu) [inline EIPS/eip-4626.md:575]: EIPs shouldn't have external links.  Please inline this code or add it to `../assets/eip-4626/` and include a relative link here.  Note: Must be CC0 licensed.

File: EIPS/eip-4626.md [hunk 2]
```diff
@@ -572,7 +572,7 @@ For production implementations of Vaults which do not use ERC-4626, wrapper adap
...
 a minimal and opinionated implementation of the standard with hooks for developers to easily insert custom logic into deposits and withdrawals.
 
 See [Vyper ERC4626](https://github.com/fubuloubu/ERC4626):
```
Comment (Pandapip1) [inline EIPS/eip-4626.md:578]: Also needs to be added to the assets folder

**Review by Pandapip1 — APPROVED** [review]
I think we've agreed that EIPW doesn't apply to Final EIPs. While I'm mildly against having external links, I'm strongly for merging PRs as fast as possible.

CC @SamWilsn

**Comment by SamWilsn** [comment]
I'd rather we just remove the external link instead of updating it.

**Comment by Pandapip1** [comment]
> I'd rather we just remove the external link instead of updating it.

I'd rather the repository be copied into the assets folder

File: EIPS/eip-4626.md [hunk 3]
```diff
@@ -576,7 +576,7 @@ For production implementations of Vaults which do not use EIP-4626, wrapper adap
 
 ## Reference Implementation
 
-See [Solmate EIP-4626](https://github.com/Rari-Capital/solmate/blob/main/src/mixins/ERC4626.sol):
+See [Solmate ERC4626](https://github.com/transmissions11/solmate/blob/main/src/mixins/ERC4626.sol):
```
[the remark this answers is not in the document]
Comment (transmissions11) [reply EIPS/eip-4626.md:579]: why

[the remark this answers is not in the document]
Comment (xinbenlv) [reply EIPS/eip-4626.md:579]: I think you are referencing to a specific an implementation of a ERC interface, instead an ERC interface itself. Referencing this file with a `.sol` postfix make it more obvious.

Comment (xinbenlv) [inline EIPS/eip-4626.md:579]: Same here.

**Pull request closed without merging**